test/browser/http.test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+define(['ably', 'shared_helper', 'chai'], function (Ably, helper, chai) {
+  var rest;
+  var expect = chai.expect;
+
+  describe('rest/http/fetch', function () {
+    this.timeout(60 * 1000);
+    before(function (done) {
+      Ably.Rest.Platform.Config.xhrSupported = false;
+      Ably.Rest.Platform.Config.jsonpSupported = false;
+      helper.setupApp(function () {
+        rest = helper.AblyRest({
+          agents: {
+            'custom-agent': '0.1.2',
```

Comment:
what's the reason we need a custom agent here?

Reply:
I think this is where we initilise `rest` client, and this is a part of `agents` test. The one I was talking about upgrading.

Reply:
https://github.com/ably/ably-js/pull/1019/commits/0fc4266904b141da76644494589867220afc53a0